spec/features/item_show_page_spec.rb

```diff
@@ -28,4 +28,89 @@
       expect(page).to have_button("Add item")
     end
   end
+
+  context 'when i visit the items show page as an unregistered user' do
+    it 'can click link to add item to cart' do
+
+      item_1 = create(:item)
+      item_2 = create(:item)
+      item_3 = create(:item)
+      item_4 = create(:item)
+
+      visit item_path(item_1.id)
+
+      expect(page).to have_button("Add item")
+
+      click_on "Add item"
+
+      expect(current_path).to eq(items_path)
+
+      within '#nav' do
+
+        expect(page).to have_content("Cart: 1")
+
+      end
+
+      expect(page).to have_content(item_1.name)
```

Comment:
Not sure that testing for the items index page to have all the items is really necessary here - we do that elsewhere right? Might be superfluous for this test.

Reply:
Noted and removed.